### DISCUSSION TITLE
Why are some treesitter syntaxes commented out?

### DISCUSSION THREAD

**Discussion by yutkat** [EdenEast/nightfox.nvim#411] [Q&A] [answered]
Before the treesitter breaking change, there was color in these things, but after the update, there is no color anymore.
https://github.com/EdenEast/nightfox.nvim/blob/7e9487875dc5f69a2fd6f60d3a3ef4fb457b57c1/lua/nightfox/group/modules/treesitter.lua#L102-L115

**Comment by EdenEast** [comment, marked as answer]
Resolved in https://github.com/EdenEast/nightfox.nvim/commit/d4339e5b30a17b7130bbbc5de3bb6407239c9f9f.

**Reply by yutkat** [reply]
Thank you very much.
